<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="view">
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="filterBox">
      <property name="title">
       <string>Filter</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="buttonUseItemTypeFilter">
         <property name="text">
          <string>Item type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="listItemTypeFilter">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="buttonUsePlaceFilter">
         <property name="text">
          <string>Place</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="listPlaceFilter">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="buttonUseWrittenOffFilter">
         <property name="text">
          <string>Written off</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="listWrittenOffFilter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Only present</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Only written off</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionOpenDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionHideFilter"/>
    <addaction name="actionPrintCSV"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionAddMultiline"/>
   </widget>
   <widget class="QMenu" name="menuItems">
    <property name="title">
     <string>Items</string>
    </property>
    <addaction name="actionShowHistory"/>
   </widget>
   <widget class="QMenu" name="menuColumns">
    <property name="title">
     <string>Columns</string>
    </property>
    <addaction name="actionShowItemType"/>
    <addaction name="actionShowItemPlace"/>
    <addaction name="actionShowResponsiblePerson"/>
    <addaction name="actionShowItemName"/>
    <addaction name="actionShowINN"/>
    <addaction name="actionShowWritingOff"/>
    <addaction name="actionShowRepairState"/>
    <addaction name="actionShowCheckedItemState"/>
    <addaction name="actionShowNote"/>
    <addaction name="separator"/>
    <addaction name="actionSwitchCheckedOff"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuEdit"/>
   <addaction name="menuItems"/>
   <addaction name="menuColumns"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionShowHistory"/>
   <addaction name="actionPrintCSV"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionUndo"/>
   <addaction name="actionAddMultiline"/>
   <addaction name="separator"/>
   <addaction name="actionHideFilter"/>
  </widget>
  <action name="actionShowHistory">
   <property name="text">
    <string>Show history...</string>
   </property>
   <property name="toolTip">
    <string>Show history</string>
   </property>
  </action>
  <action name="actionAddMultiline">
   <property name="text">
    <string>Add multiline...</string>
   </property>
  </action>
  <action name="actionPrintCSV">
   <property name="text">
    <string>Print CSV...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionHideFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide filter</string>
   </property>
   <property name="toolTip">
    <string>Hide filter</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionShowItemType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Item type</string>
   </property>
   <property name="toolTip">
    <string>Show item type</string>
   </property>
  </action>
  <action name="actionShowItemPlace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Item place</string>
   </property>
   <property name="toolTip">
    <string>Show item place</string>
   </property>
  </action>
  <action name="actionShowResponsiblePerson">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Responsible person</string>
   </property>
   <property name="toolTip">
    <string>Show responsible person</string>
   </property>
  </action>
  <action name="actionShowItemName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Item name</string>
   </property>
   <property name="toolTip">
    <string>Show item name</string>
   </property>
  </action>
  <action name="actionShowINN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>INN</string>
   </property>
   <property name="toolTip">
    <string>Show INN</string>
   </property>
  </action>
  <action name="actionShowWritingOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Writing off</string>
   </property>
   <property name="toolTip">
    <string>Show writing off</string>
   </property>
  </action>
  <action name="actionShowRepairState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repair state</string>
   </property>
   <property name="toolTip">
    <string>Show repair state</string>
   </property>
  </action>
  <action name="actionShowCheckedItemState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Checked item state</string>
   </property>
   <property name="toolTip">
    <string>Show checked item state</string>
   </property>
  </action>
  <action name="actionShowNote">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
   <property name="toolTip">
    <string>Show note</string>
   </property>
  </action>
  <action name="actionSwitchCheckedOff">
   <property name="text">
    <string>Switch &quot;Checked&quot; off</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open database...</string>
   </property>
   <property name="toolTip">
    <string>Open database file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHideFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUseItemTypeFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>listItemTypeFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUsePlaceFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>listPlaceFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUseWrittenOffFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWrittenOffFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
